hack/extra-values.yaml

```diff
@@ -1,5 +1,14 @@
 auth:
-  secretName: aws-creds
-  accessKeyId: ""
-  secretAccessKey: ""
-  sessionToken: ""
\ No newline at end of file
+  # Leave secret undefined for implicit auth (node instance role, IMDSv2, etc.)
+  secret: {}
+    # secretName: aws-creds
+    # accessKeyId: ""
+    # secretAccessKey: ""
+    # sessionToken: ""
+    # By default, a secret will be created. Leave the above fields blank and specify 'createSecret: false' to use an existing secret.
+    # WARNING: the existing secret must match the format used in auth-secret.yaml
+    # createSecret: true
+
+  # Override the service account used by AWS validator (optional, could be used for IMDSv2 on EKS)
+  # WARNING: the chosen service account must have the same RBAC privileges as seen in manager-rbac.yaml
+  serviceAccountName: ""
```

Comment:
whats the reason for having this `extra-values.yaml`? it looks like all the values that can be defined here could also be defined in the more common `values.yaml`

Reply:
It's a holdover from when I was first autogenerating the chart. There's a make target that appends this file to the autogenerated values.yaml. I should probably just remove it now.